### DISCUSSION TITLE
How to remove repositories that I starred from my feed?

### DISCUSSION THREAD

**Discussion by asdfMaciej** [community/community#67004] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't want to follow starred repos - I give out stars as a token of appreciation, without any intentions to follow

**Comment by baggiponte** [comment]
Totally second this - I star repos to keep a personal "second brain" of project, and I don't really need to see the updates of each one of them.

**Comment by hafiz-muhammad** [comment]
Example:
![2023-09-13_10-29](https://github.com/community/community/assets/109435420/57363a18-a20e-4b84-becd-4c363ada537d)
![2023-09-13_10-30](https://github.com/community/community/assets/109435420/52de0cdb-8386-4451-a5d0-fd148f838269)
After that refresh your page.

**Reply by Sauceke** [reply]
This is not a viable workaround as you'd have to do this for each repo you've ever starred individually.

**Reply by joakime** [reply]
Totally agree, this should be a global configuration.

**Comment by justinclift** [comment, marked as answer]
There is a browser extension to restore the old feed: https://github.com/SamJakob/github-homepage

There's also a userscript style approach that apparently does the same thing (haven't tried this one though): https://github.com/Gerrit0/old-github-feed
